mycoffee/params.py

```diff
@@ -188,3 +189,10 @@
         "info": "Steep-and-release method"
     }
 }
+
+COFFEE_UNITS_MAP = {
+    "g": {"name": "gram", "rate": 1},
+    "oz": {"name": "ounce", "rate": 0.03527396195},
+    "lb": {"name": "pound", "rate": 0.00220462262185},
+    "mg": {"name": "milligram", "rate": 1000},
+    "kg": {"name": "kilogram", "rate": 0.001}}
```

Comment:
```
"kg": {"name": "kilogram", "rate": 0.001},
}
```

Reply:
fixed in https://github.com/sepandhaghighi/mycoffee/pull/33/commits/444b4eb206da89feefbc60e2b2606129ac2c05fa